docs/doc/10-deploy/09-upgrade/10-compatibility.md

```diff
@@ -107,6 +107,18 @@ The following is an illustration of current query-meta compatibility:
 <img src="/img/deploy/compatibility.excalidraw.png"/>
 
 
+# Compatibility between different versions of databend-meta
```

Comment:
Maybe we can improve this chapter title as `## Compatibility between different versions of databend-meta`, so we can toggle it as title? Now the toggle title is not good:
<img width="309" alt="image" src="https://github.com/datafuselabs/databend-docs/assets/172204/0cbfd80e-873f-498f-9bb9-be2751a47d3d">

Reply:
<img width="505" alt="image" src="https://github.com/datafuselabs/databend-docs/assets/44069/19642b43-5c0f-4e72-93d2-04b0e0ac97ec">